controllers/githubHooks.js

```diff
@@ -28,7 +28,7 @@ var HooksController = {
       }
 
       // Begin the webhook decoding
-      var body = JSON.parse(req.body.payload);
```

Comment:
I assume it gets parsed by `app.use(bodyParser.json())` in `app.js`?

I don't know how this code worked in the first place, though.